Reemit.Gui/ViewModels/Controls/HexEditor/HexEditorNavigationViewModel.cs

```diff
@@ -0,0 +1,122 @@
+﻿using AvaloniaHex.Document;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+using Reemit.Gui.ViewModels.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+
+namespace Reemit.Gui.ViewModels.Controls.HexEditor;
+
+public class HexEditorNavigationViewModel : ReactiveObject
+{
+    private readonly HexEditorViewModel _hexEditorViewModel;
+
+    [Reactive]
+    public BitRange? NavigationBitRange { get; set; }
+
+    [ObservableAsProperty]
+    public HexNavigationRangeViewModel? ResolvedNavigationRange { get; set; }
```

Comment:
Looks like the setter is redundant here.

Reply:
Oops. Fixed.